src/FM.LiveSwitch.Mux.Standard/MuxOptions.cs

```diff
@@ -34,6 +34,9 @@ public class MuxOptions
         [Option('f', "frame-rate", Default = 30, HelpText = "The frames per second of the output video.")]
         public int FrameRate { get; set; }
 
+        [Option("min-orphan-duration", Default = 120, HelpText = "Minimum duration (in minutes) of waiting for incomplete media file to have file size unchanged for the media to be considered orphan.")]
```

Comment:
Minor language correction:
`"Minimum duration (in minutes) of waiting for incomplete media files to have file size unchanged for the media to be considered orphaned."`